agent/tests/test_php_wrapper.c

```diff
@@ -181,14 +181,47 @@ static void setup_nested_framework_calls(nrspecialfn_t one_before,
 
 /*
  * to customize txn naming per framework, PHP agent uses the order in which
-functions are processed, NR_NOT_OK_TO_OVERWRITE/NR_OK_TO_OVERWRITE, and whether
-it is called either before or after NR_PHP_WRAPPER_CALL (for pre PHP 8+) or
-whether it is called in func_begin or func_end (for PHP 8+ / OAPI).
+ * (functions are processed, NR_NOT_OK_TO_OVERWRITE/NR_OK_TO_OVERWRITE, and
+ * whether it is called either before or after NR_PHP_WRAPPER_CALL (for pre PHP
+ * 8+) or whether it is called in func_begin or func_end (for PHP 8+ / OAPI).
  *
  * This test serves to illustrate and test *framework* txn naming conventions
-and how they are affected by calling nr_tx_set path either 1) before
-nr_wrapper_call for preoapi and in func_begin as a before callback in oapi 2)
-after nr_wrapper_call for preoapi and in func_end as an after callback in oapi
+ * and how they are affected by calling nr_tx_set path either 1) before
+ * nr_wrapper_call for preoapi and in func_begin as a before callback in oapi 2)
+ * after nr_wrapper_call for preoapi and in func_end as an after callback in
+ * oapi
+ *
+ * The execution order is as follows:
+ * 1) oapi: `one` before callback OR legacy:any statements before the `one`
+ * PHP_CALL_WRAPPER
+ *
+ *     2) `one` begins execution and calls `two`
+ *
+ *         3) oapi: `two` before callback OR legacy: any statements before the
+ * `two` PHP_CALL_WRAPPER
+ *
+ *              4) `two` begins execution and calls `three`
+ *
+ *                  5) oapi: `three` before callback OR legacy: any statements
+ * before the `three` PHP_CALL_WRAPPER
+ *
+ *                      6) `three` begins execution
+ *
+ *                      7) `three` ends
+ *
+ *                   8) oapi: `three` after callback OR legacy:
+ * any statements after the `three` PHP_CALL_WRAPPER
+ *
+ *               9) `two` ends
+ *
+ *          10) oapi:
+ * `two` after callback OR legacy: any statements after the `two`
+ * PHP_CALL_WRAPPER
+ *
+ *      11) `one` ends
+ *
+ * 11) oapi: `one` after callback OR legacy: any statements after the `one`
+ * PHP_CALL_WRAPPER
```

Comment:
If `// clang-format off` is added, add matching `// clang-format on` to re-enable formatting.

Reply:
Thanks for the idea about putting the call stack, it made sense to include it to show the flow.
clang-format was fine with the block and didn't touch the formatting.